scripts/validate-schemas/validate-schemas.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2024 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"cuelang.org/go/cue/load"
+	"github.com/perses/perses-plugins/scripts/npm"
+	"github.com/sirupsen/logrus"
+)
+
+const schemasPath = "backend"
```

Comment:
why not calling the folder `schema` instead ? backend for me doesn't define the place where the schema should be defined

Reply:
well I called it that way because it's technically the backend part of the plugin. I dont want to change the structure of the frontend side while gabriel is working on it but I was thinking that in the end a plugin could look like
```
MyPlugin/
- backend/
  - cue.mod/
  - mypluginschema.cue
  - migrate.cue
- frontend
  - package.json
  - src/
  - ...
```
Or maybe that's unnecessary?

Reply:
well the thing is, the day we'll be ready to implement the plugin installation part, schemas & frontend code need to be installed in different places probably? that's why I was thinking to have wrapper folders here to separate them, to hopefully make this easier.

Reply:
Actually, in the current (new) implementation of the plugin management in Perses, schema & frontend can be in the same place and it will make sense they are in the same places actually

Reply:
Tackled offline, this is actually not how the plugin files were planned to be organized, let's change that in another PR since this is not related to that one